### DISCUSSION TITLE
Inbox permanently indicates unread messages

### DISCUSSION THREAD

**Discussion by PaluMacil** [community/community#120435] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

The little blue indicator on the inbox icon always shows, so I cannot tell if I have unread messages without manually changing pages to view my inbox. On the left I see 1 message indicated in the lefthand pane for mentioned and participating, but both are empty. I can click either an still see no massages in the main area. Switching to all, unread, at the top clicking the inbox icon in the top left (which indicates 1) all result in showing no messages. Sometimes the paging counter indicates 1-0 of 1.

Has anyone else been able to clear this issue after experiencing it? It causes a lot of annoyance and flow interruption. I did place a support ticket, but I'm not sure it'll get much attention if they are dealing with more important bugs.

**Comment by PaluMacil** [comment, marked as answer]
Resolved: Support responded instantly. They escalated it to engineering and let me know I can take care of it myself even before a fix via the notifications API. Message follows:

In the interim, the curl command below will mark all of your notifications as read, clear the notification indicator, and with any luck, take care of that phantom message for you. 
 
A few things to note:
In order to use this command a Personal Access Token (PAT) will be required and `$TOKEN` needs to be replaced by a personal access token you can create here: https://github.com/settings/tokens/new. [Creating a Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token#creating-a-token) also has the steps to create a token. After it is created, replace `$TOKEN` in the command with your new PAT.

Note: Please ensure that the PAT has the `notifications` scope.

Please update the date and time to your current date and note the time values must be in ISO 8601 format: `YYYY-MM-DDTHH:MM:SSZ`.

```
curl -H "Authorization: bearer $TOKEN" -X PUT -H "Accept: application/vnd.github.v3+json" https://api.github.com/notifications -d '{"last_read_at":"2022-11-08T13:21:00"}'

```
You can also export the current date and time in bash by running:

```
export now=$(date +%Y-%m-%dT%H:%M:%SZ)

```
Then you can use the date and time variable in the curl command as follows:

```
curl -H "Authorization: bearer $TOKEN" -X PUT -H "Accept: application/vnd.github.v3+json" https://api.github.com/notifications -d '{"last_read_at":"'"$now"'"}'

```
For your reference, here is the documentation for the [Notifications REST API endpoint](https://docs.github.com/en/rest/reference/activity#mark-notifications-as-read).

**Comment by MarvinXu** [comment]
<img width="1012" height="690" alt="image" src="https://github.com/user-attachments/assets/78532aee-b22a-441d-ad16-7f53b710df28" />
Is this bug fixed?

**Reply by Zooce** [reply]
I have this exact issue - same repo.

**Reply by SharakPL** [reply]
Same for me with many different repos (I don't think that's even relevant which repo it is). I've had "Inbox: 1" showing for years now with completely no idea what that 1 stands for. My inbox currently has hundreds of unread notifications 😅
